### DISCUSSION TITLE
[Feature] .insp Photos stitching (Insta360)

### DISCUSSION THREAD

**Discussion by rafaelsroman** [immich-app/immich#12194] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently .insp Pictures from the Insta360x4 in my case are not stitched together correctly see in the attached screenshot:

![image_2024-08-31_184444811](https://github.com/user-attachments/assets/813e85a1-d1fb-44e6-be2d-5773af7bad06)

I have also found this older thread where this issue was talked about https://github.com/immich-app/immich/pull/2725#issuecomment-1913104041

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Handkep** [comment]
Yes. .dng files are being shown als plain images.
Also: would it be possible to get 360° video straight out of the insta cameras to work?

**Comment by ullmannJan** [comment]
Yes support for .insp .insv and .lrv files would be great!
